### PR TITLE
fix hook_feed_fetched signature, allow install via git clone

### PR DESCRIPTION
**Pull request by shoeper**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -11,6 +11,6 @@ checkdeps:
 
 install: checkdeps
 	install -d /usr/share/tt-rss/www/plugins/ff_xmllint
-	install -m 644 ff_xmllint/init.php /usr/share/tt-rss/www/plugins/ff_xmllint/
+	install -m 644 init.php /usr/share/tt-rss/www/plugins/ff_xmllint/
```
Comment (fastcat) [inline Makefile:14]: the Makefile is meant to run from the repo root, and in that case the `init.php` is very definitely in the `ff_xmllint` subdirectory.

For this change to work, you would need to move everything out of the subdirectory, which isn't in this PR. Did you miss committing or pushing a bit?

**Review by fastcat — CHANGES_REQUESTED** [review]
Thanks for the contribution! There seems to be a bit missing from the PR however. If you can clean that up, I'm happy to merge this :)

File: Makefile (hunk 1 above)
Comment (shoeper) [reply Makefile:14]: I thought I also did that. Not sure why it is missing.

The goal was to allow install from git, which works best when everything is in the root folder.

Comment (shoeper) [reply Makefile:14]: Maybe I forgot to push the change. Now, the move should be there.

Comment (fastcat) [reply Makefile:14]: still isn't pushed

**Pull request merged**